barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover.cpp

```diff
@@ -154,17 +154,10 @@ template <class ProverInstances> void ProtoGalaxyProver_<ProverInstances>::prepa
         send_accumulator(instance, domain_separator);
     } else {
         // This is the first round of folding and we need to generate some gate challenges.
-        // TODO(https://github.com/AztecProtocol/barretenberg/issues/740): implement option 2 to make this more
-        // efficient by avoiding the computation of the perturbator
         finalise_and_send_instance(instance, domain_separator);
         instance->target_sum = 0;
-        auto beta = transcript->template get_challenge<FF>(domain_separator + "_initial_gate_challenge");
-        std::vector<FF> gate_challenges(instance->log_instance_size);
-        gate_challenges[0] = beta;
-        for (size_t i = 1; i < instance->log_instance_size; i++) {
-            gate_challenges[i] = gate_challenges[i - 1].sqr();
-        }
-        instance->gate_challenges = gate_challenges;
+        instance->gate_challenges.resize(instance->log_instance_size);
+        std::fill(instance->gate_challenges.begin(), instance->gate_challenges.end(), 0);
```

Comment:
```suggestion
        instance->gate_challenges = std::vector<FF>(instance->log_instance_size,0);
```
i thinkn this works as well